grcs/0020-knowledge-graph.md

```diff
@@ -0,0 +1,527 @@
+---
+GRC: 20
```

Comment:
nit: could you use quotes, i.e. `"20"`? otherwise markdown parsing breaks sometimes